Makefile

```diff
@@ -690,8 +690,8 @@ release:
 	    echo "WARNING: branch $$X.$$Y already exists: you may want to run make patch instead" ;\
 	    git merge origin/master ;\
 	 else \
-	    git checkout master -b $$X.$$Y && echo zedcloud.zededa.net > conf/server &&\
-	    git commit -m"Setting default server to prod" conf/server ;\
+	    git checkout master -b $$X.$$Y && echo zedcloud.zededa.net > conf/server && sed 's/set_getty/# set_getty/g' -i conf/grub.cfg && \
+	    git commit -m"Setting default server to prod and disable getty" conf/server conf/grub.cfg ;\
```

Comment:
Don't you think use console instead of getty in commit's title would let the message (explanation) more clear?

"Setting default server to prod and disable **console**"